assets/src/components/ImageHoverControls/index.js

```diff
@@ -0,0 +1,36 @@
+import {Button} from '@wordpress/components';
+const {__} = wp.i18n;
+
+export const ImageHoverControls = props => {
+  const {
+    onEdit,
+    onRemove,
+    isCompact,
+    isAdd,
+  } = props;
+
+  return <div className="buttons-overlay">
+    { isAdd && <Button
+      onClick={onEdit}
+      icon="plus-alt2"
+      isPrimary
+      className="edit-image"
+    >
+      { __('Add image', 'planet4-blocks-backend') }
```

Comment:
We could keep them for now to avoid loosing the translations. Once we have everything over to the theme we can explore a migration process to switch the text domain while keeping the translations.